model/version.go

```diff
@@ -307,9 +307,9 @@ func VersionGetHistory(versionId string, N int) ([]Version, error) {
 }
 
 func GetVersionsWithOptions(projectName string, opts GetVersionsOptions) ([]Version, error) {
-	projectId, err := FindIdForProject(projectName)
+	projectId, err := GetIdForProject(projectName)
```

Comment:
Does it ever seem possible people would want this to work with old project names, i.e. if this errors we should still just use projectName? I'm hesitant about supporting it because probably they should know about the new project name, but it would prevent breaking scripts if something does change.

Reply:
I think what you have right now is best. Nonexistent project IDs (because they were deleted?) seem like something we don't need to handle